examples/docs_snippets/docs_snippets/concepts/partitions_schedules_sensors/sensors/sensors.py

```diff
@@ -239,8 +247,49 @@ def multi_asset_sensor(context):
     )
 
 
+# end_custom_multi_asset_sensor_marker
+
+# start_multi_asset_sensor_marker
+
+
+@multi_asset_sensor(
+    asset_keys=[AssetKey("asset_a"), AssetKey("asset_b")],
+    job=my_job,
+)
+def asset_a_and_b_sensor(context, asset_events):
+    # this sensor will run when both asset_a and asset_b have been materialized
+    # asset_events is a dictionary, mapping asset keys to event log entries
+    if all(asset_events.values()):
+        logger_str = (
+            f"Assets {asset_events[AssetKey('asset_a')].dagster_event.asset_key.path} "
+            f"and {asset_events[AssetKey('asset_b')].dagster_event.asset_key.path} materialized"
+        )
+        return RunRequest(
+            run_key=context.cursor,
+            run_config={"ops": {"logger_op": {"config": {"logger_str": logger_str}}}},
+        )
+
+
 # end_multi_asset_sensor_marker
 
+# start_multi_asset_sensor_custom_fn_marker
+
+
+@multi_asset_sensor(
+    asset_keys=[AssetKey("asset_c"), AssetKey("asset_d")],
+    job=my_job,
+)
+def asset_c_or_d_sensor(context, asset_events):
+    # this sensor will run when either asset_c or asset_d materialize
+    if any(asset_events.value()):
```

Comment:
should this be `.values`?  maybe put these under test?

Reply:
yep good catch. i'll put them all under test after i revamp the docs with the new changes